### PR TITLE
Created a capstone project on photography summit

### PR DESCRIPTION
**Pull request by lawrahkonwea**
This is my first capstone project which I personalized to a photography summit and exhibition. In this project, I created it using;
- Html
- CSS
- Javascript
-  The project has two pages, the home and the about section both in mobile and desktop screen sizes.
-  The speaker section was created dynamically using javascript and has a 'see more' button which was created as well using Javascript
-  The hamburger was created using Html, CSS, and javascript which when the about is clicked on, takes the user directly to the about section
- The home page and about page are also linked, whereby the logo takes the user to the home page and the about links to the about page as well.
- Link to the loom video https://www.loom.com/share/e1b8c7a9fc524979be408b845e61bcb4

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        <h2 class="main-program2">Partners</h2>
+        <hr class="line">
+        <div class="connect-img">
+          <img class="the-img" src="images/mozilla.png" alt="">
+          <img class="the-img" src="images/Google_grey.png" alt="">
+          <img class="the-img" src="images/kakao-2.png" alt="">
+          <img class="the-img" src="images/airbnb-2.png" alt="">
+        </div>
+      </div>
+    </section>
```
Comment (Qoosim) [inline about.html]: - [ ] This section is not supposed to be visible for the desktop version on the about page. Kindly look closely to the guiding template and work your design alongside the guiding temp. Please watch the screen record to understand what I am saying the more.

### Screen record

[your-about-footer.webm](https://user-images.githubusercontent.com/26597432/203824576-b4758354-f483-4600-8314-1ae351d74671.webm)


- [ ]  For the mobile version, it is required to stick to the guiding template as close as possible. This involves typography, color, padding, margin among others.  The height of your design for this section and the arrangement of its contents are not in line with what we have on the guiding template. Kindly look at the guiding template and work your design along that direction. Take a look.

### Screen record
[about-footer-mobile.webm](https://user-images.githubusercontent.com/26597432/203825613-a62be43b-0b7b-4a85-9706-7bc0b8506993.webm)

File: about.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        <div id="footer-con">
+          <div class="logo">
+            <a href="index.html"><img class="my-logo" src="images/my-logo.png" alt=""></a>
+          </div>
+          <div>
+            <p class="down" id="changes">2022 Summit. Some Rights Reserved.</p>
+          </div>
+        </div>
+      </div>
+    </section>
```
Comment (Qoosim) [inline about.html]: - [ ] The footer background color on the guiding design is black but on your own design is white. Kindly change your footer background color to black.

### Your Design
your-about-footer-bg

### Guiding Design
about-footer-bg-color

File: index.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        <div id="footer-con">
+          <div class="logo">
+            <a href="index.html"><img class="my-logo" src="images/my-logo.png" alt=""></a>
+          </div>
+          <div>
+            <p class="down" id="changes">2022 Summit. Some Rights Reserved.</p>
+          </div>
+        </div>
+      </div>
+    </section>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required to produce a close replica of the guiding template. This includes color, font, background-color, margin, padding among others. The height of both the partner and footer sections on the guiding template are the same, but your design looks totally different from that. Kindly look into it and fix it.

[your-home-footer-page.webm](https://user-images.githubusercontent.com/26597432/203830317-26c7e003-ab50-4812-bd10-e621e593de96.webm)

File: index.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+const seemore = document.getElementById('viewmore');
+seemore.addEventListener('click', () => {
+  wholeSpeakers.classList.toggle('activated');
+  if (seemore.textContent === 'see more speakers') {
+    seemore.textContent = 'see less';
+  } else {
+    seemore.textContent = 'see more speakers';
+  }
```
Comment (Qoosim) [inline index.js:78]: - [ ] Kindly add more styling to this section for it to look like the guiding template. Take a look

[speaker-section.webm](https://user-images.githubusercontent.com/26597432/203831029-6245ff73-281d-46b2-9a30-55c14de373d5.webm)

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @lawrahkonwea, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi @lawrahkonwea , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**